### DISCUSSION TITLE
is  this UI responsive?

### DISCUSSION THREAD

**Discussion by laoshaw** [tusen-ai/naive-ui#1782] [General]
can not find this very basic info from the documentation, also what about browser compatibility

**Comment by laoshaw** [comment]
https://www.naiveui.com/en-US/light/components/grid  to answer my own question, it does seem support responsive grid.

**Reply by 07akioni** [reply]
Not much components support it, but grid does.

**Comment by nullsub** [comment]
Is responsive support planned in the future?

**Reply by 07akioni** [reply]
How would you expect the responsive be?

Is there any specific example?

**Reply by nullsub** [reply]
I have problems with the DataTable. This is an example from https://www.naiveui.com/en-US/os-theme/components/data-table:
![datatable](https://user-images.githubusercontent.com/328502/149475667-80c3654c-72d8-4174-b413-ce696e7416a8.jpg)

I would expect it to somehow horizontally scrollable and each entry be shown in one line

Thank you for your help

**Reply by 07akioni** [reply]
Just put an overflow auto div outside table in mobile mode. That's what I did in document site API table.

**Reply by nullsub** [reply]
Thank you, I also added "white-space: pre;" to the whole table. Now it looks perfect!
